hr_employee_operating_unit/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2017 Genweb2 Limited - Matiar Rahman
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from . import models
+from . import tests
```

Comment:
Importing tests is not needed

Reply:
Hi @aheficent ,

I fixed that. Thanks.